### PR TITLE
moved buffered_read to inline function

### PR DESCRIPTION
**Pull request by anshul1912**
Signed-off-by: Anshul Maheshwari <er.anshul.maheshwari@gmail.com>

### CODE REVIEW

File: src/lib_ccx/general_loop.c [hunk 1]
```diff
@@ -264,7 +264,7 @@ int general_getmoredata(struct lib_ccx_ctx *ctx, struct demuxer_data **data)
 	do
 	{
 		want = (int) (BUFSIZE - ptr->len);
-		buffered_read (ctx->demux_ctx, ptr->buffer + ptr->len, want); // This is a macro.
+		ressult = buffered_read (ctx->demux_ctx, ptr->buffer + ptr->len, want); // This is a macro.
```
Comment (xvkdev) [inline src/lib_ccx/general_loop.c]: This extra 's' in result doesn't look good

Comment (anshul1912) [inline src/lib_ccx/general_loop.c]: thanks and corrected in later commit

**Comment by anshul1912** [comment]
@ccx-testerbot runtests master

**Comment by canihavesomecoffee** [comment]
@ccx-testerbot runtests master

**Pull request merged**